provisioning/device/src/Transports/Mqtt/ProvisioningTransportHandlerMqtt.cs

```diff
@@ -185,7 +185,7 @@ protected override void Dispose(bool disposing)
             X509Certificate2 clientCertificate = ((AuthenticationProviderX509)message.Authentication).GetAuthenticationCertificate();
 
             var tlsSettings = new ClientTlsSettings(
-                TlsVersions.Instance.Preferred,
+                this.SslProtocols,
```

Comment:
We don't use `this` unless absolutely necessary, and there's almost always a way to avoid using it.

```suggestion
                SslProtocols,
```